Chapter3/Chapter3.Task1/Program.cs

```diff
@@ -11,9 +11,10 @@ static void Main(string[] args)
         {
             Console.WriteLine("MyFirstTask");
             String firstName = "FirstMyTask";
-            Console.WriteLine("firstName has {0} characters.", FirstName.Length);
-            Console.ReadLine();
 
+            Console.WriteLine("firstName has {0} characters.", firstName.Length);
```

Comment:
I expect you to improve existing implementation. E.g. task says to read string from console, you - hard coded it. Please read from console the string and improve messages to user and result in order to make them more informative